internal/nodes/avi_model_l7_translator.go

```diff
@@ -693,3 +697,20 @@ func buildPoolWithInfraSetting(key string, pool *AviPoolNode, infraSetting *akov
 		utils.AviLog.Debugf("key: %s, msg: Applied AviInfraSetting configuration over PoolNode %s", key, pool.Name)
 	}
 }
+
+func buildListenerPortsWithInfraSetting(infraSetting *akov1alpha1.AviInfraSetting, aviPortProto []AviPortHostProtocol) {
```

Comment:
@arihantg: we need to add validation about `enableSSL` same as that present in Hostrule.

Reply:
@akshayhavile It will mean we will reject any aviinfrasetting object containing listeners if all the ports specified are ssl disabled.
Currently in hostrule we need at least one port with sslEnabled to accept the hostrule object.

Reply:
yes